src/main/java/Duke/TaskList.java

```diff
@@ -14,55 +14,55 @@ public TaskList() {
         this.taskList = new ArrayList<>();
         this.numTasks = 0;
     }
-    public String processTask(Task task) {
+    public String processTask(Task task){
         this.taskList.add(task);
-        this.numTasks += 1;
+        this.numTasks +=1;
```

Comment:
Small nit, should have a whitespace between `+=` and  `1`. I've seen a lot of whitespace violations in your code. 

You can refer to https://se-education.org/guides/conventions/java/intermediate.html#layout , under the `White space within a statement` section